pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -108,7 +151,9 @@ func (s *NodeController) Run(period time.Duration, syncNodeList, syncNodeStatus
 			glog.Errorf("Error loading initial static nodes: %v", err)
 		}
 	}
+	nodes = s.DoChecks(nodes)
 	nodes, err = s.PopulateAddresses(nodes)
+	nodes = reconcilePodCIDRs(nodes, &api.NodeList{})
```

Comment:
It is awkward right now, because we assign CIDRs to nodes, but we only configure them with that CIDR if they are running in a cloud provider (and only in GCE right now). It is probably harmless, but it would be easier to follow if we only called reconcileCIDRs() when we are going to call configureCIDR().